### ISSUE TITLE
[Masthead/Footer] Accessibility - The element with "complementary" role does not have a label - DDS Consulting

### ISSUE THREAD

**Issue by SR1947** [carbon-design-system/carbon-for-ibm-dotcom#6784]
IBM Equal Access Accessibility Checker
The element with "complementary" role does not have a label
<dds-legal-nav slot="legal-nav" data-autoid="dds--footer-legal-nav" role="complementary">

----
- Hybrid Cloud developer: 
- [HC Jira ticket]
- Dev estimate:  
- DDS consultant:

**Comment by RobertaJHahn** [comment]
@annawen1 @guilhermelMoraes Take a screen shot of where the complementary roll is.  Reach out to Juan Leon to fix, if needed.

**Comment by RobertaJHahn** [comment]
@Praveen-sr Could you have Ram take a look at this issue and provide a severity?  Thanks.

**Comment by annawen1** [comment]
> 
The element with the "complementary" role in the Footer would be the legal-nav container

<img width="1581" alt="Screen Shot 2021-08-03 at 9 01 54 AM" src="https://user-images.githubusercontent.com/54281166/128021158-767ea705-2230-4c91-89f7-fcd311639d80.png">

**Comment by proeung** [comment]
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2150
HC Engineer Assignee: @andy-blum
